### PR TITLE
subsys/mgmt/hawkbit: change 'Hawkbit' & 'hawkbit' to 'hawkBit'

### PR DESCRIPTION
**Pull request by ycsin**
Change 'Hawkbit' and 'hawkbit' to 'hawkBit' wherever makes sense.

[preview](https://builds.zephyrproject.io/zephyr/pr/71057/docs/samples/subsys/mgmt/hawkbit/README.html)

### CODE REVIEW

**Review by maass-hamburg — DISMISSED** [review]
LGTM

File: samples/subsys/mgmt/hawkbit/README.rst [hunk 1]
```diff
@@ -25,22 +25,22 @@ source code for this mode hawkbit_manual
 Caveats
 *******
 
-* The Zephyr port of ``Hawkbit`` is configured to run on a
+* The Zephyr port of ``hawkBit`` is configured to run on a
```
Comment (kartben) [inline samples/subsys/mgmt/hawkbit/README.rst:28]: I know you only did the search replace, but I don't think it makes sense to use double quotes here (and several other instances in this file)

image

File: samples/subsys/mgmt/hawkbit/prj.conf [hunk 2]
```diff
@@ -18,7 +18,7 @@ CONFIG_BOOTLOADER_MCUBOOT=y
 #Main Stack Size
 CONFIG_MAIN_STACK_SIZE=4096
 
-#Minimal Heap mem pool size for the Hawkbit working
+#Minimal Heap mem pool size for the hawkBit working
```
Comment (kartben) [inline samples/subsys/mgmt/hawkbit/prj.conf:21]:
```suggestion
#Minimal Heap mem pool size for hawkBit to work
```

File: subsys/mgmt/hawkbit/Kconfig [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # SPDX -License-Identifier: Apache-2.0
 
 menuconfig HAWKBIT
-	bool "Hawkbit Firmware Over-the-Air support"
+	bool "hawkBit Firmware Over-the-Air support"
```
Comment (kartben) [inline subsys/mgmt/hawkbit/Kconfig:5]: albeit a bit tricky, let's try to comply with the Eclipse Foundation Trademark Usage Policy and change this one as follows:

```suggestion
	bool "Eclipse hawkBit Firmware Over-the-Air support"
```

**Review by kartben — CHANGES_REQUESTED** [review]
thanks for doing this!

File: samples/subsys/mgmt/hawkbit/prj.conf [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#Turn on the Hawkbit
+#Turn on the hawkBit
```
Comment (kartben) [inline samples/subsys/mgmt/hawkbit/prj.conf:1]:
```suggestion
#Turn on hawkBit
```

File: subsys/mgmt/hawkbit/hawkbit_priv.h [hunk 5]
```diff
@@ -7,7 +7,7 @@
 /** @file
  *
  * @brief This file contains structures representing JSON messages
- * exchanged with a hawkbit
+ * exchanged with a hawkBit
```
Comment (kartben) [inline subsys/mgmt/hawkbit/hawkbit_priv.h:10]:
```suggestion
 * exchanged with a hawkBit server
```

**Comment by ycsin** [comment]
Thanks @kartben, good catch!

**Review by maass-hamburg — DISMISSED** [review]

File: samples/subsys/mgmt/hawkbit/README.rst (hunk 1 above)
Comment (kartben) [reply samples/subsys/mgmt/hawkbit/README.rst:28]: @ycsin would be great to address this as part of the clean-up?

File: subsys/mgmt/hawkbit/hawkbit.c [hunk 6]
```diff
@@ -10,7 +10,7 @@
 
 #include <zephyr/logging/log.h>
 
-LOG_MODULE_REGISTER(hawkbit, CONFIG_HAWKBIT_LOG_LEVEL);
+LOG_MODULE_REGISTER(hawkBit, CONFIG_HAWKBIT_LOG_LEVEL);
```
[the remark this answers is not in the document]
Comment (ycsin) [reply subsys/mgmt/hawkbit/hawkbit.c:13]: sure, I also forgot to remove the double tick, will do another rev

File: samples/subsys/mgmt/hawkbit/README.rst (hunk 1 above)
Comment (ycsin) [reply samples/subsys/mgmt/hawkbit/README.rst:28]: Yeah I forgot about this, will do another rev

**Comment by ycsin** [comment]
@kartben I also made some changes to the 'frdm_k64f', LMK what you think, thanks

**Review by maass-hamburg — APPROVED** [review]

**Review by kartben — APPROVED** [review]
nice - thanks @ycsin!

**Pull request merged**